notebooks/CED_spikeinterface_pipeline.ipynb

```diff
@@ -104,18 +80,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "recording = se.CEDRecordingExtractor(ced_file, smrx_ch_inds=rhd_channels)"
+    "recording = se.CEDRecordingExtractor(ced_file, smrx_channel_ids=rhd_channels)\n",
+    "\n",
+    "prb_channels = [\n",
+    "    42, 36, 32, 38, 39, 47, 45, 37, 44, 46, 48, 35, 34, 40, 41, 43, 62, 60, 58, 56, 54, 52, 50, 33, 49, 51, 53,\n",
+    "    55, 57, 59, 61, 63, 1, 3, 5, 7, 9, 11, 13, 30, 14, 12, 10, 8, 6, 4, 2, 0, 21, 27, 31, 25, 24, 16, 18, 26,\n",
+    "    19, 17, 15, 28, 29, 23, 22, 20\n",
+    "]\n",
+    "prb_locations = [[0, x * 20] for x in range(len(prb_channels))]\n",
+    "recording.set_channel_locations(locations=np.array(prb_locations)[prb_channels, :])"
```

Comment:
We should provide a test file instead. I added it to the repo, but it's not there anymore. I'll make a PR with the cambridge neurotech probe and tetrodes probes

Reply:
Actually, the probe file requested for these kind of CED files is in main: https://github.com/catalystneuro/mease-lab-to-nwb/blob/main/probe_files/cambridge_neurotech_H3.prb